posthog/models/property.py

```diff
@@ -82,15 +81,7 @@ class BehaviouralPropertyType(str, Enum):
 
 VALIDATE_BEHAVIOURAL_PROP_TYPES = {
     BehaviouralPropertyType.PERFORMED_EVENT: ["key", "value", "event_type", "time_value", "time_interval"],
-    BehaviouralPropertyType.PERFORMED_EVENT_MULTIPLE: [
-        "key",
-        "value",
-        "event_type",
-        "time_value",
-        "time_interval",
-        "operator",
-        "operator_value",
```

Comment:
Are there defaults in place? The way the condition is written, `operator` can't be none

Reply:
Indeed: https://github.com/PostHog/posthog/blob/cohorts-new-model/ee/clickhouse/models/cohort.py#L145, None means `=`.

Reply:
For operator value though?

Reply:
that should still be in here, indeed!